### ISSUE TITLE
VGGFace2 as FR dataset

### ISSUE THREAD

**Issue by jihyeonyi** [openvinotoolkit/datumaro#77]
VGGFace2 is basically FR (Face recognition) dataset, 
but our implementation only supports landmark and bounding box annotations.
I think we should add Label(...) so it can be used for FR task.

And many fr data has the same structure except the landmark information.
That is, a directory represents a person and it contains images for the person.
So if we make landmark annotations to be optional, we can support lots of FR data.

In summary, 
1. we need Label(...) annotations
2. (optional) I suggest making the landmark/bbox annotations to be optional.

**Comment by zhiltsov-max** [comment]
Hi, if I understand right, you're suggesting: 
- to consider directory names as labels
- to produce labels, when no bbox information available

How about reading in `ImageNet` format (at least, as a temporary solution)? You can also apply `anns_to_labels` transform to create a classification dataset from a detection one.

**Comment by jihyeonyi** [comment]
Yes, you're right.
I've tried `ImageNet` format, but it requires an additional filename format. 
So, I can't read VGGFace2 data with it.
Thank you for suggesting `anns_to_labels`. I'll check it later.
